projects/ngx-rxtimeline/src/lib/line/line.component.ts

```diff
@@ -10,6 +10,13 @@ import { Line } from '../core/line';
       [attr.y2]="line.y2"
     ></svg:line>
   `,
+  styles: [
+    `
+      line {
```

Comment:
Maybe this should go in the ngx-rxtimeline.component.scss? ViewEncapsulation for the root component is set to none so I don't think this style will be localised to this component.

Reply:
yep, overlooked that we were doing it all that the top level.